proprietary/design-tokens/src/components/utrecht/paragraph.tokens.json

```diff
@@ -10,7 +10,7 @@
       "margin-block-end": { "value": "{rvo.space.md}" },
       "lead": {
         "color": {},
-        "font-size": {},
+        "font-size": { "value": "{rvo.font-size.xl}" },
         "font-weight": {},
         "line-height": {}
```

Comment:
Looking at the existing code, I would expect the font size to be `lg` and not `xl`.

https://github.com/nl-design-system/rvo/blob/main/documentation/demopages/common/style.scss#L59-L62

Why xl?

Reply:
That's a mistake and is now fixed. It doesn't look like changes to the tokens update in Storybook. Also, missing the line-height, also now added.